CameraXBasic/app/build.gradle

```diff
@@ -36,6 +36,10 @@ android {
         targetCompatibility rootProject.ext.java_version
     }
 
+    kotlinOptions {
+        jvmTarget = JavaVersion.VERSION_1_8
```

Comment:
Can you please use the versioned varaible? I.e. "$rootProject.ext.java_version"

Reply:
Done.